README.md

```diff
@@ -1 +1,50 @@
-# Leaderboard
\ No newline at end of file
+# Leaderboard
+
```

Comment:
Please add `npm install` and `npm start` commands in README as the project can not be set up locally without these commands.